src/main/java/avd/inf/jdm/rentmycar/controller/dto/UserDto.java

```diff
@@ -13,16 +13,27 @@
 @Setter
 public class UserDto {
     @NotNull
```

Comment:
Doordat het id @NotNull is moet je het id altijd meegeven in de body van de request. In principe is dit niet nodig aangezien je deze ook al in de call meegeeft `/users/1`. Ik stel voor deze @NotNull ook weg te halen

Reply:
done!